src/flexPlugin/index.ts

```diff
@@ -85,11 +85,15 @@ class FlexPluginProvider implements pulumi.dynamic.ResourceProvider {
     }
 
     public async delete(id:pulumi.ID, props: any) {
-
+        
         const client = getTwilioClient();
 
         await getAPI(client, ["serverless", { "services" : props.serviceSid }, "environments" ])(id).remove();
```

Comment:
We should remove these lines and let the Flex Plugins API handle that. As @TheGarkine mentioned in the PR comments, with these lines some times the delete is failing and I suspect that this is happening because we are doing this change manually following the old process and the Flex Plugins API probably use a new process.